tests/run/mixin-primitive-on-generic-1.scala

```diff
@@ -0,0 +1,19 @@
+
+object Test {
+  def main(args: Array[String]): Unit = {
+    println((new Foo: Baz).value1)
+    println((new Foo: Baz).value2)
+  }
+}
+
+class Foo extends Bar[Boolean](true) with Baz
```

Comment:
Done in `mixin-primitive-on-generic-4.scala`and `mixin-primitive-on-generic-5.scala` and I checked that the method called is the unboxed one.